### PR TITLE
wrong path for install file

### PR DESCRIPTION
**Pull request by mdeguzis**
This also affects Krypton. After view, please merge into the Krypton branch as well.

### CODE REVIEW

File: debian/kodi-audioencoder-vorbis.install [hunk 1]
```diff
@@ -1,3 +1,2 @@
-usr/lib/*/addons/audioencoder.vorbis/audioencoder.vorbis.so* /usr/lib/kodi/addons/audioencoder.vorbis
+usr/lib/*/kodi/addons/audioencoder.vorbis/audioencoder.vorbis.so* /usr/lib/kodi/addons/audioencoder.vorbis
```
Comment (wsnipex) [inline debian/kodi-audioencoder-vorbis.install]: install path is wrong.
please make it:
usr/lib
usr/share

**Comment by mdeguzis** [comment]
That's strange, shows right with [patch-4](https://github.com/mdeguzis/audioencoder.vorbis/blob/patch-4/debian/kodi-audioencoder-vorbis.install).

**Comment by mdeguzis** [comment]
Squashed commits and made change.

**Comment by mdeguzis** [comment]
@wsnipex , like with all the pvr addons, the install path is directly pointing to /usr/lib/kodi, when it should be pointing to /usr/lib/<multiarch>/kodi. See the latest commit change. This is why in testing hts, the lib wasn't being picked up in Kodi, as it was not in /usr/lib/<multiarch>/kodi. Chopping the destination off allows it to be installed in the proper place.

**Pull request merged**